### DISCUSSION TITLE
Presa controllo consumi

### DISCUSSION THREAD

**Discussion by jovannig** [indomus/forum#2013] [Hardware & Firmware]
Ho alcune prese smart Teckin configurati su tuya /smartlife e portati su HA tramite l'integrazione tuya.
Non riesco pero' a veder i consumi, posso gestire solo l'on /off.
C'e' un modo per farlo?
Oppure ci sono prese smart compatibili HA con cui posso gestire i consumi?

**Comment by Scialla** [comment]
Io ho le vecchie SP22 tasmotate (ancora col vecchio chip) se hai il nuovo potresti guardare qui:
https://github.com/arendst/Tasmota/issues/3950

Penso che con l'integrazione Tuya, al momento, la misurazione non sia possibile

**Reply by jovannig** [reply]
Anch'io ho le SP22, ma non sono pratico con la tasmotizzazione. E' complicato e richiede una modifica hardware?

**Reply by Scialla** [reply]
no, nessuna modifica, ma con  i nuovi chip è più complicato.. non hai letto la discussione che ti ho linkato vero?

**Reply by jovannig** [reply]
Si l'ho letta ma non e' semplice da capire per me che non sono esperto.

**Comment by 50494554524F** [comment]
prova con local tuya

https://github.com/rospogrigio/localtuya

(se non va manco con questa non saprei, io ho delle aunics con solo consumo istantaneo)

**Comment by jovannig** [comment]
<img width="762" alt="image" src="https://user-images.githubusercontent.com/16450789/197872541-fd758fb0-76d5-4653-bd77-86e12029f7f3.png">

Quando arrivo a questa schermata non so come andare avanti. 
Ho provato in tanti modi ma non riesco mai a creare l'entita' relativa al consumo istantaneo.

**Comment by jovannig** [comment]
Alla fine sono riuscito a configurarlo, grazie!
